src/docs/cursors.dox

```diff
@@ -72,6 +72,26 @@ well as any key and value.
 
 See @ref transactions for more information.
 
+@section cursors_eviction Cursors and Eviction
+
+Cursor positions hold resources that may inhibit the eviction of memory pages.
+While eviction can occur as the result of reaching a threshold of cache
+memory usage (see @ref_single tune_cache), it may also occur when an
+individual memory page grows to be larger than the configured \c
+memory_page_max (see the configuration in WT_SESSION::create).
+
+If a cursor is active on a page being considered for eviction, the eviction
+will defer until the cursor is moved. In the specific cases with multiple
+active cursors on a memory page that has reached maximum size, the
+WiredTiger library will never have simultaneous control over both cursors to
+move them to smaller, split pages.  This may prevent a large page from ever
+being split and may potentially slow down the application.
+
+To avoid this situation, or any situation where eviction must be deferred,
+applications should call WT_CURSOR::reset during times that it does not need
+to keep a cursor positioned.  A cursor that has been reset is not active and
+cannot inhibit eviction.
+
```

Comment:
@ddanderson, can we skip the details of eviction? That is, does it matter what kind of eviction is being inhibited by cursors? It seems like unnecessary detail to me, and likely to lead to the wording becoming incorrect as the system evolves.